src/RobotInterface/include/BipedalLocomotion/RobotInterface/ISensorBridge.h

```diff
@@ -0,0 +1,373 @@
+/**
+ * @file ISensorBridge.h
+ * @authors Prashanth Ramadoss
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_ISENSOR_BRIDGE_H
+#define BIPEDAL_LOCOMOTION_ROBOT_INTERFACE_ISENSOR_BRIDGE_H
+
+#include <memory>
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+#include <Eigen/Dense>
+
+#include <BipedalLocomotion/GenericContainer/Vector.h>
```

Comment:
This one is not required here, right?

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/87/commits/699ea3f560fc27118e23d9aa30f562487e0ef527